src/main/java/com/minecolonies/coremod/colony/managers/CitizenManager.java

```diff
@@ -346,6 +350,11 @@ public void setMaxCitizens(final int newMaxCitizens)
         this.maxCitizens = newMaxCitizens;
     }
 
+    /**
+     * Datas about the happiness of a colony
+     */
+    private final HappinessData happinessData = new HappinessData();
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Move this variable to comply with Java Code Conventions. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.minecolonies.com/coding_rules#rule_key=squid%3AS1213)

Reply:
Please this